app/uk/gov/hmrc/eoricommoncomponent/frontend/DateConverter.scala

```diff
@@ -26,11 +29,28 @@ object DateConverter {
 
   private val logger = Logger(this.getClass)
 
+  val earliestYearDateOfBirth         = 1900
+  val earliestYearEffectiveVatDate    = 1970
+  val earliestYearDateOfEstablishment = 1000
```

Comment:
Do we have a confirmation about those years? Are they correct?

I remember discussion about date of birth, but rest of them not

Reply:
Dates were discussed on Slack with Steve.   But I'll add these dates to the other "date" ticket (ECC-279) and get Product team to confirm.  We can then change them if necessary.